packages/extensions/core/resources/plugin-csharp.md

```diff
@@ -38,7 +38,7 @@ use-extension:
 
 ##### Input API versions (azure-rest-api-specs + C# specific)
 
-``` yaml $(csharp) && !isRequested('@autorest/csharp')
+``` yaml $(csharp) && !isRequested('@autorest/csharp') && ($(legacy) || $(v2))
```

Comment:
Should we remove all the `isRequested(???)` if they don't work anyway?

Reply:
yeah so it seems like this is another bug. If you explicitly pass `use` it should not have run this config regardless of --v2 or --legacy.

Reply:
yeah, this doesn't seem like an easy fix so I am going to add that to the list of tests to verify for the configuration refactor.